src/schema/rules/sidecars/asl.yaml

```diff
@@ -51,12 +53,20 @@ MRIASLTextOnlySliceTiming:
                     final entry in the `SliceTiming` list is the time of acquisition of slice 0.
                     Without this parameter slice time correction will not be possible.
 
-MRIASLTextOnlyFlipAngle:
+MRIASLTextOnlyFlipAngleRec:
+    selectors:
+    - modality == "MRI"
+    - datatype == "perf"
+    - suffix == "asl"
+    - '!(LookLocker in sidecar) || sidecar.LookLocker == false'
```

Comment:
@tsalo how do you feel about this? or would you prefer two separate rules?

Reply:
I like having one rule, and this is pretty readable.